x-pack/plugins/cases/server/routes/api/user_actions/get_all_user_actions.test.ts

```diff
@@ -6,16 +6,30 @@
  */
 
 import { getUserActionsRoute } from './get_all_user_actions';
+import { docLinksServiceMock } from '@kbn/core/server/mocks';
 
 describe('getUserActionsRoute', () => {
+  const docLinks = docLinksServiceMock.createSetupContract();
+
   it('marks the endpoint internal in serverless', async () => {
-    const router = getUserActionsRoute({ isServerless: true });
+    const router = getUserActionsRoute({ isServerless: true, docLinks });
 
     expect(router.routerOptions?.access).toBe('internal');
+    expect(router.routerOptions?.deprecated).toMatchInlineSnapshot(`
+      Object {
+        "documentationUrl": "https://www.elastic.co/guide/en/kibana/test-branch/breaking-changes-summary.html#breaking-201004",
```

Comment:
>        "documentationUrl": "https://www.elastic.co/guide/en/kibana/test-branch/breaking-changes-summary.html#breaking-201004",

If it's at all possible to make this a fuzzier match that would be a good idea IMO since the initial part of that URL is almost certain to change as we migrate to the new docs system.

Reply:
We will test that the url ends with `#breaking-201004`